src/components/CheckBox/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { borderRadiuses, colors } from "@common/styleVariables";
+import styled, { CSSObject } from "@emotion/styled";
+import { DetailedHTMLProps, InputHTMLAttributes } from "react";
+
+const errorStyle: CSSObject = {
+	borderColor: colors.error,
+};
+
+const StyledCheckbox = styled.input<Props>(({ error, disabled, readOnly }) => ({
+	appearance: "none",
+	margin: 0,
+	marginLeft: "0.5em",
+	verticalAlign: "middle",
+	width: "1.2em",
+	height: "1.2em",
+	border: `1px solid ${colors.mediumContrast}`,
+	borderRadius: borderRadiuses.small,
+	backgroundColor: colors.white,
+	cursor: "pointer",
+	"&:checked": {
+		backgroundColor: colors.blueDark,
+	},
+	"&:hover": !disabled &&
+		!readOnly && {
+			boxShadow: `0px 0px 0px 2px ${colors.neutral300}`,
+		},
+	"&:disabled": {
+		backgroundColor: colors.neutral200,
+		cursor: "not-allowed",
+	},
+	...(error ? errorStyle : {}),
+}));
+
+export interface Props extends DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement> {
+	error?: string | null;
+}
+
+export default function Checkbox(props: Props) {
+	const { type = "checkbox", ...otherProps } = props;
+	return <StyledCheckbox type={type} {...otherProps} />;
```

Comment:
Hier sollten wir nur `"checkbox"` als type erlauben – andere Types machen hier ja keinen Sinn.

```suggestion
	return <StyledCheckbox type="checkbox" {...props} />;
```

Reply:
Soweit angepasst. Hier würde ich dann deine gebaute CheckBox nutzen. Daher erstmal "halbfertig" ohne Haken.

Reply:
Vielleicht baust du sie direkt ein und machst dann einen merge?